api/src/main/resources/api/service_openapi.yaml

```diff
@@ -1237,11 +1250,25 @@ components:
 
     InstanceV2:
       type: object
-      description: A map of entity attribute names to their value for this instance
-      additionalProperties:
-        type: object
-        nullable: true
-        $ref: "#/components/schemas/ValueDisplayV2"
+      properties:
+        attributes:
+          description: A map of entity attribute names to their value for this instance
+          type: object
+          additionalProperties:
+            type: object
+            nullable: true
+            $ref: "#/components/schemas/ValueDisplayV2"
+        hierarchyFields:
+          description: Hierarchy field values for this instance
+          type: object
+          properties:
+            path:
+              type: string
+              nullable: true
```

Comment:
Kind of weird that only this field is nullable instead of returning a default value like the other two.

Reply:
Agreed it's weird. Not sure how best to address this.

If `path` has a null value, then it's not part of the hierarchy. That's the condition the UI uses to determine whether to show the hierarchy icon, right?

How about adding another hierarchy field to handle this instead of overloading `path`? e.g. `isMember` is true if the instance is part of the hierarchy, `path` is no longer nullable.

Reply:
I took a stab at adding this `isMember` property PTAL.

Reply:
That's right. I had forgotten an empty paths wasn't the same as a non-existent one.

I like this better anyways. It's more obvious what it means.